ios/RecaptchaEnterpriseReactNative.swift

```diff
@@ -28,6 +28,29 @@ class RecaptchaEnterpriseReactNative: NSObject {
     }
   }
 
+  @objc(fetchClient:withResolver:withRejecter:)
+  func fetchClient(
+    siteKey: String,
+    resolve: @escaping RCTPromiseResolveBlock,
+    reject: @escaping RCTPromiseRejectBlock
+  ) {
+    let getClientClosure: (RecaptchaClient?, Error?) -> Void = { recaptchaClient, error in
```

Comment:
fetchClientClosure?

Reply:
Refactored it to reuse code between APIs.